commcare_connect/connect_id_client/main.py

```diff
@@ -70,6 +70,12 @@ def filter_users(country_code: str, credential: list[str]):
     return [ConnectIdUser(**user_dict) for user_dict in data["found_users"]]
 
 
+def fetch_user_counts() -> dict[str, int]:
```

Comment:
We should cache this

Reply:
Also, it doesn't seem very ideal to fetch this data from CID, is there no way to infer it on CCC?

Reply:
There is not a way to infer it on CCC, but I agree we should cache it. Honestly, we could just save it in postgres. We only need to fetch the current month (and maybe the last one if we haven't fetched since the month switched). We might even be able to do that nightly on a task so loading the table never causes a network call.